apps/channels/tests/test_base_channel_behavior.py

```diff
@@ -42,7 +42,7 @@ def test_incoming_message_adds_channel_info(_get_llm_response, _send_text_to_use
     _simulate_user_message(telegram_channel, message)
 
     experiment_session = ExperimentSession.objects.filter(
-        experiment=telegram_channel.experiment, external_chat_id=chat_id
+        experiment=telegram_channel.experiment, participant__external_chat_id=chat_id
```

Comment:
what do you think about making a method on the manager for this filter since it's quite common and long:

`ExperimentSession.objects.for_external_id(chat_id, experiment)`

Reply:
Not a bad idea :+1: